src/components/Patient/Utils.ts

```diff
@@ -17,3 +23,23 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: { patientId: string }) => {
+  const queryClient = useQueryClient();
+  const { patientId } = options;
+
+  return useMutation({
+    mutationFn: mutate(routes.addPatientNote, {
+      pathParams: { patientId },
+    }),
+    onSuccess: (data) => {
+      console.log("here");
+      queryClient.invalidateQueries({
+        queryKey: ["notes", patientId],
+        exact: false,
+      });
+      Notification.Success({ msg: "Note added successfully" });
```

Comment:
let's skip the notification for success.

user will be aware that it's added by seeing the message move up to the message history.